src/styles/theme.ts

```diff
@@ -57,7 +57,7 @@ const initTheme = (darkMode: boolean) => {
     },
     spacing: base,
     shape: {
-      borderRadius: '6px',
```

Comment:
Nit: I think we should keep this for consistency (I initially thought it would use theme spacing).

Reply:
I only changed the value to be `6` instead of `6px` because there were errors in the console.